src/index.js

```diff
@@ -209,6 +216,10 @@ export class BrowserHistory extends History {
       this.options.routeHandler(fragment) :
       false;
   }
+
+  _createOrigin(protocol: string, hostname: string, port: string) {
```

Comment:
Yes, but I guess I don't see much value in testing the string concatenation. I'd be more concerned with testing the public method to verify that we're actually returning the expected value, given some combinations of location and history root.